tests/server/disco/TestViews.js

```diff
@@ -51,11 +54,14 @@ describe('Discovery Pane GET requests', () => {
     .then((res) => {
       assert.equal(res.header.location,
         '/en-US/firefox/discovery/pane/48.0/Darwin/normal');
+      return true;
     }));
 
   it('should set an HSTS header', () => request(app)
```

Comment:
It is, yeah.

I would love to stop using implicit returns. I *really* dislike their syntax and I also dislike that they make our code feel less homogeneous, especially because the eslint rule *requires* them if they can be used.